R/av.R

```diff
@@ -28,7 +28,7 @@ NULL
             msg <- content$response$text
     } else if (nzchar(type) && grepl("text/html", type)) {
         ## these pages can be too long for a standard 'stop()' message
-        cat(as.character(response), file = stderr())
+        cat(as.character(response), fill = TRUE, file = stderr())
```

Comment:
I don't think this is appropriate -- these will be long HTML documents that probably have structure but do not wrap well to, e.g., an 80-column display.